src/app/services/blockchain.service.ts

```diff
@@ -109,10 +111,12 @@ export class BlockchainService {
       .map((status: any) => {
         if (!status.connections || status.connections.length === 0) {
           this.onLoadBlockchainError(ConnectionError.NO_ACTIVE_CONNECTIONS);
-          return null;
+          throw { reported: true };
         }
 
         return status;
-      });
+      })
+      .flatMap(() => this.apiService.get('version'))
```

Comment:
this wont happen if the exception above is thrown, but I guess it's ok since the nodeVersion is not used anywhere yet

Reply:
At this time, when the node does not have connections to other nodes, an error message is displayed and the entire wallet is stopped, so `nodeVersion` would not be needed (even after starting to use it). The wallet needs something to allow checking the node status again, but in any case the error should be very rare.